components/Assets/AssetListDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { Feather } from '@expo/vector-icons';
+import { View } from 'react-native';
+import { Token } from 'types/supabaseTypes';
+
+import { FText } from '../Text/FText';
+
+interface AssetListDisplayProps {
+  token: Token;
+}
+
+export const AssetListDisplay = ({ token }: AssetListDisplayProps) => {
+  // const actualValue = token.value[-1].value;
+
+  const actualValue = 1234.56;
```

Comment:
Do we want to keep the placeholder data?
Since we didn't in the graph maybe we don't here, just to keep it the same?